blackjax/adaptation/mclmc_adaptation.py

```diff
@@ -30,10 +30,13 @@ class MCLMCAdaptationState(NamedTuple):
         The momentum decoherent rate for the MCLMC algorithm.
     step_size
         The step size used for the MCLMC algorithm.
+    std_mat
```

Comment:
hmmm could we use more informative naming? Is this the diagonal of the covariance matrix? Maybe `diag_cov`?

Reply:
sqrt of it, so the diagonal of the standard deviation matrix. But yes, I can update the name